scm/driver/github/util.go

```diff
@@ -22,6 +23,33 @@ func encodeListOptions(opts scm.ListOptions) string {
 	return params.Encode()
 }
 
+func formRepoListQueryParams(opts scm.RepoListOptions) string {
```

Comment:
Can we keep the name consistent to `encodeRepoListOptions` for each provider?

Reply:
For github we are not encoding the query params like we are doing for the other providers, hence different name.
When we encode the query params and try the same out in the curls and through the code, it doesnt work, hence no encoding query params for github

Reply:
Okay can you once check for gitlab ?

Reply:
for gitlab, in the existing List api, we are calling, `encodeMemberListOptions` hence in the ListV2 api we are calling `encodeMemberRepoListOptions` keeping the naming inline with the existing names